docs/COMPONENT_WORKFLOW.md

```
@@ -26,9 +27,11 @@ ls packages/ui/src/components/ui/
 Before creating a custom component, check if shadcn/ui has it:
 
 **Browse available components:**
+
 - https://ui.shadcn.com/docs/components
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)